### ISSUE TITLE
Connection between 2 systems

### ISSUE THREAD

**Issue by souravrane** [siddharthasahu/P2P-chat-application#1]
Is there a way to connect 2 systems over the internet ? if yes , what ips do we provide in the set and add friend boxes ?

**Comment by sdht0** [comment]
You can connect to any system that you can directly access via an ip and port.

**Comment by souravrane** [comment]
okay ! but the question is how ? i tried out many possible inputs and still did not work out well. 
Please tell me what is it that is to be added in the respective fields so that 2 different systems connected by internet can converse.

**Comment by sdht0** [comment]
It is not enough that the 2 systems are connected to the internet. The 2 systems should be **directly** accessible via an ip:port. So the systems would need to have a public ip, for example.

**Comment by ajish097** [comment]
The number of messages that gets displayed once you connect couple of peers are also changing. Do they get removed or  the peer threads not synchronized ?

**Comment by sdht0** [comment]
Yeah, they might be bugs in the code. This is a basic implementation that I got working, so your mileage may vary.

**Comment by marcelozocchio** [comment]
Hello, Siddharta! I added AES CFB mode encryption to your chat application. See if it looks good:
https://github.com/sdht0/P2P-chat-application/compare/master...marcelozocchio:marcelozocchio-patch?expand=1
